app/directives/ui/datatables/registry-repositories-datatable/registryRepositoriesDatatable.html

```diff
@@ -0,0 +1,113 @@
+            <i class="fa fa-search" aria-hidden="true"></i> Search
+          </span>
+        </div>
+      </div>
+      <!--<div class="actionBar">
+        <button type="button" class="btn btn-sm btn-danger"
+          ng-disabled="$ctrl.state.selectedItemCount === 0" ng-click="$ctrl.removeAction($ctrl.state.selectedItems)">
+          <i class="fa fa-trash space-right" aria-hidden="true"></i>Remove
+        </button>
+        <button type="button" class="btn btn-sm btn-primary" ui-sref="actions.create.registry">
+          <i class="fa fa-plus space-right" aria-hidden="true"></i>Add registry
+        </button>
+      </div>-->
+      <div class="searchBar" ng-if="$ctrl.state.displayTextFilter">
+        <i class="fa fa-search searchIcon" aria-hidden="true"></i>
+        <input type="text" class="searchInput" ng-model="$ctrl.state.textFilter" placeholder="Search...">
+      </div>
+      <div class="table-responsive">
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <th>
+                <span class="md-checkbox">
+                  <input id="select_all" type="checkbox" ng-model="$ctrl.state.selectAll" ng-change="$ctrl.selectAll()" />
+                  <label for="select_all"></label>
+                </span>
+                <a ng-click="$ctrl.changeOrderBy('name')">
+                  Name
+                  <i class="fa fa-sort-alpha-asc" aria-hidden="true" ng-if="$ctrl.state.orderBy === 'name' && !$ctrl.state.reverseOrder"></i>
+                  <i class="fa fa-sort-alpha-desc" aria-hidden="true" ng-if="$ctrl.state.orderBy === 'name' && $ctrl.state.reverseOrder"></i>
+                </a>
+              </th>
+              <th>
+                Tags Count
+              </th>
+              <!--<th>
+                <a ng-click="$ctrl.changeOrderBy('URL')">
+                  URL
+                  <i class="fa fa-sort-alpha-asc" aria-hidden="true" ng-if="$ctrl.state.orderBy === 'URL' && !$ctrl.state.reverseOrder"></i>
+                  <i class="fa fa-sort-alpha-desc" aria-hidden="true" ng-if="$ctrl.state.orderBy === 'URL' && $ctrl.state.reverseOrder"></i>
+                </a>
+              </th>
+              <th>Actions</th>-->
+            </tr>
+          </thead>
+          <tbody>
+            <tr dir-paginate="item in ($ctrl.state.filteredDataSet = ($ctrl.dataset | filter:$ctrl.state.textFilter | orderBy:$ctrl.state.orderBy:$ctrl.state.reverseOrder | itemsPerPage: $ctrl.state.paginatedItemLimit))" ng-class="{active: item.Checked}">
+              <td>
+                <span class="md-checkbox">
+                  <input id="select_{{ $index }}" type="checkbox" ng-model="item.Checked" ng-change="$ctrl.selectItem(item)"/>
+                  <label for="select_{{ $index }}"></label>
+                </span>
+                <a ui-sref="registry.tags({id: registry.Id, repository: item.name})" ui-sref-opts="{inherit: false}">{{ item.name }}</a>
```

Comment:
To fix the state transition here:

* Use `<a ui-sref="registry.tags({id: $ctrl.registryId, repository: item.name})">{{ item.name }}</a>` to define the link
* Add a new binding to `registryRepositories.js`:

```
registryId: '<'
```

* Bind the registryId with the datatable in `registryBrowse.html`:

```
<div class="row">
  <div class="col-sm-12">
    <registry-repositories-datatable
    title="Repositories" title-icon="fa-database"
    dataset="repositories" table-key="repositories"
    order-by="name" show-text-filter="true"
    access-management="applicationState.application.authentication"
    remove-action="removeAction"
    registry-id="registry.Id"
    ></registry-repositories-datatable>
  </div>
</div>
```